package.json

```diff
@@ -44,6 +47,7 @@
     "object.entries": "1.0.4",
     "object.values": "1.0.4",
     "page": "1.7.1",
+    "probability-distributions": "^0.9.1",
```

Comment:
It slightly pains me to import a ton of code just to use one function, but then again it's tiny compared to the overall size of BBGM..